packages/desktop-ui-lib/src/lib/time-tracker/timer-session/timer-session.component.html

```
@@ -0,0 +1,102 @@
+				[isSingleDatePicker]="true"
+				[isDisableFutureDatePicker]="true"
+				class="medium"
+				(rangeChanges)="onRangeChange($event)"
+				[dates]="selectedRange"
+			></ngx-date-range-picker>
+		</div>
+	</div>
+
+	<div class="filter-bar">
+		@for (f of filterOptions; track f.value) {
+			<button class="filter-chip" [class.active]="statusFilter() === f.value" (click)="setFilter(f.value)">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)